src/v2/util/isTouchDevice.ts

```diff
@@ -0,0 +1,4 @@
+// Extracted from https://stackoverflow.com/posts/4819886/revisions
+export function isTouchDevice() {
+  return 'ontouchstart' in window || navigator.maxTouchPoints
+}
```

Comment:
https://github.com/aredotna/ervell/blob/master/src/v2/util/is.js#L13

Reply:
Thanks, will remove.